src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/tests/latest/test_acs_commands.py

```diff
@@ -5,31 +5,43 @@
 
 import os
 import tempfile
+import unittest
 
 from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer,
                                RoleBasedServicePrincipalPreparer)
 
 # flake8: noqa
 
+AZURE_TEST_RUN_LIVE = bool(os.environ.get('AZURE_TEST_RUN_LIVE'))
+
+
+@unittest.skip("needs explicit service principal management")
```

Comment:
This skips this test class in both playback and live. You're sure this is what you want?

Reply:
Yes--it's really a work-in-progress now. We need to use explicit SP management so this doesn't leak one each time it runs, but `az acs create --orchestrator-type=Kubernetes` doesn't support this.

I can remove it entirely if that would be cleaner.